kebechet/managers/thoth_advise/messages.py

```diff
@@ -0,0 +1,86 @@
+
+DEFAULT_PR_BODY = (
+    """Automatic update of Pipfile.lock by Kebechet thoth-advise manager."""
+)
+
+MISSING_PACKAGE_VERSION_PR_BODY = """
+Version {version} of {package} has gone missing from {index}.  This package is currently used in your project.
+
+Thoth has automatically run `advise` on your repository as a result. This PR contains an automatic update of \
+Pipfile.lock by Kebechet thoth-advise manager.
+"""
+
+MISSING_PACKAGE_VERSION_ISSUE_BODY = """
+Version {version} of {package} has gone missing from {index}. This package is currently used by your project.
+
+This has caused your application stack to become unsolvable. Please loosen your version constraints for {package}.
+"""
+
+NEW_CVE_PR_BODY = """
+Project Thoth registered new CVE for version {version} of {package}. This package is currently used in your project.
+
+Thoth has automatically run `advise` on your repository as a result. This PR contains an automatic update of \
+Pipfile.lock by Kebechet thoth-advise manager.
+"""
+
+NEW_CVE_ISSUE_BODY = """
+Project Thoth registered new CVE for version {version} of {package}. This package is currently used by your project.
+
+This has caused your application stack to become unsolvable. Please loosen your version constraints for {package} or \
+change your recommendation type.
+"""
+
+MISSING_PACKAGE_PR_BODY = """
+{package} has gone missing from {index}. This package was being used by your project as a transitive dependency.
+
+Thoth was able to find versions of dependencies to avoid using {package} in the locked application stack. This PR \
+contains an automatic update of Pipfile.lock by Kebechet thoth-advise manager.
+"""
+
+MISSING_PACKAGE_ISSUE_BODY = """
+{package} has gone missing from {index}. This package is currently used in your project. This issue is critical and \
+will result in failed builds.
+
+Please find another index which hosts this package or refactor to use a different library.
+"""
+
+HASH_MISMATCH_PR_BODY = """
+Version {version} of {package} on {index} has hashes which no longer match those stored in Thoth's database. This \
+indicates that the package maintainers likely added or removed an artifact (binary).
+
+Thoth has automatically run `advise` on your repository as a result. This PR contains an automatic update of \
+Pipfile.lock by Kebechet thoth-advise manager.
+"""
+
+HASH_MISMATCH_ISSUE_BODY = """
+Version {version} of {package} on {index} has hashes which no longer match those stored in Thoth's database. This \
+indicates that package maintainers likely removed an artifact (binary).
+
+This has caused your application stack to become unsolvable. Please loosen version constraints or change your runtime \
+environment.
+"""
+
+NEW_RELEASE_PR_BODY = """
+Version {version} of {package} was released. {package} is used in your project so Thoth automatically ran advise on \
+your application stack.
+
+This PR contains an automatic update of Pipfile.lock by Kebechet thoth-advise manager.
```

Comment:
Should we divide this based on the recommendation used?

if user requested `latest` -> make sense to update 
if user requested `performance` -> make sense if the new stack gives more performance
if user requested `security` -> ??
if user requested `stable` -> ??

wdyt?

Reply:
We can't guarantee that the new package is included in the update

Reply:
> > We can't guarantee that the new package is included in the update
> 
> I think we can provide pipeline units in adviser that read metadata supplied (e.g. which package was newly released) and will make sure these packages are part of the lock file (if found any). Then, the overall score will decide if the lock file is better than the one already used by the user.

I think make sense, and if the score is lower than the previous one, we should not open PR right?

Reply:
+1, yes.